src/Hive.IO/Hive.IO/Forms/Controls/SurfaceTechnologyProperties.cs

```diff
@@ -29,6 +29,17 @@ public override ConversionTechPropertiesViewModel Conversion
             }
         }
 
+        /// <summary>
+        ///     See [here](https://stackoverflow.com/a/3165330/2260) for why I'm not
+        ///     hooking the event up directly with the handler defined in the base class.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private new void Validating(object sender, CancelEventArgs e)
+        {
+            TextBox_Validating(sender, e);
+        }
+
```

Comment:
not sure if the problem comes from here, but see [this comment](https://github.com/architecture-building-systems/hive/pull/666#issuecomment-963412851)